testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -25,18 +25,52 @@
 from numpy.testing import assert_equal
 
 from MDAnalysis.lib._cutil import (
-    unique_int_1d, find_fragments, _in2d,
+    unique_int_1d,
+    find_fragments,
+    _in2d,
 )
 
 
-@pytest.mark.parametrize('values', (
-    [],  # empty array
-    [1, 1, 1, 1, ],  # all identical
-    [2, 3, 5, 7, ],  # all different, monotonic
-    [5, 2, 7, 3, ],  # all different, non-monotonic
-    [1, 2, 2, 4, 4, 6, ],  # duplicates, monotonic
-    [1, 2, 2, 6, 4, 4, ],  # duplicates, non-monotonic
-))
```

Comment:
To me here it makes sense again to keep the old formatting. The new one is _really_ ugly.

Reply:
Yes, I think at this point of the review I still hadn't realized the behavior of trailing commas.